static/src/javascripts/bower.json

```diff
@@ -6,7 +6,7 @@
     "domready": "ded/domready#v0.3.0",
     "eventEmitter": "~4.2.6",
     "fastdom": "0.8.5",
-    "lodash-amd": "~2.4.1",
+    "lodash-amd": "regiskuckaertz/lodash-amd#2.4.2",
```

Comment:
As discussed at lunch, this is the line that worries me most. I see two solutions

1. Fork into `guardian/lodash-amd#2.4.2` at least if you leave, someone else will be able to maintain that fork
1. Use `lodash 4`. It's modular so you can include only the bits you need without wasting too many bytes

the second approach could be used as a starter in the lodash migration. Last time it didn't work because we were changing the whole website at once, which come with a huge risk. Using two versions at the same time we can migrate files at a slower pace.